### PR TITLE
fix(tooltip): position and focus outline fixes

### LINKED ISSUE #2142: v10: Click-to-open tooltip: Visual issue with "on top" option
From @shixiedesign at IBM/carbon-components-react#2047:

- [ ] Tooltip's shark fin / triangle tip for Left, Top, and Right are all a couple pixels off 
image 
image
image

Reproducing this problem requires the following steps:

1. Go to https://the-carbon-components.netlify.com/component/tooltip--default.html
2. Type the following in Chrome DevTools' console: `document.querySelector('[data-floating-menu-direction]').setAttribute('data-floating-menu-direction', 'top')`
3. Click on the trigger icon

### PR DESCRIPTION
**Pull request by asudoh**
Fixes #2142.
Fixes IBM/carbon-components-react#2048.
Refs IBM/carbon-components-react#2061.

#### Changelog

**Changed**

- Fix for tooltip positioning in left/top/right positioning.
- Fix for focus outline color in "no icon" option.

#### Testing / Reviewing

Testing requires the following (both can be done in Netlify link):

* For positioning:
  1. Follow "view full render" link in tooltip demo
  2. Do the following in Chrome DevTools' console: `document.querySelector('[data-floating-menu-direction]').setAttribute('data-floating-menu-direction', 'top')`
  3. Click on the trigger icon
* For focus outline color in "no icon" option:
  1. Follow "view full render" link in tooltip demo
  2. Do the following in Chrome DevTools' console: `document.querySelector('.bx--tooltip__label').setAttribute('tabindex', '0')`
  3. Click on the tooltip label

### CODE REVIEW

**Review by emyarod — APPROVED** [review]
tooltip positioning and label focus outline look good to me

**Review by tw15egan — APPROVED** [review]
Looks good to me, ready to merge once @IBM/carbon-designers sign off 👍

**Review by aagonzales — APPROVED** [review]
Perfect! Thanks for the instructions on how to see it.

**Pull request merged**
